### DISCUSSION TITLE
JFormField for folders and file listing on server side

### DISCUSSION THREAD

**Discussion by bcordis** [Joomla-Bible-Study/Proclaim#926] [Ideas]
Update system to allow for server side listing of file and folders under the drop-down lists.

**Comment by genu** [comment]
Is this referring to when the user is adding a media file? Which drop-down lists is this talkin about?

**Comment by bcordis** [comment]
Yes I'm thinking this could populate the field based on if it is a local
folder.

On Friday, April 5, 2013, Eugen Istoc wrote:

> Is this referring to when the user is adding a media file? Which drop-down
> lists is this talkin about?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Joomla-Bible-Study/Joomla-Bible-Study/issues/71#issuecomment-15952672
> .

## 

Thanks,
Brent Cordis
2406 Hermitage Park Dr.
Hermitage, TN 37076
615.429.6426

**Comment by genu** [comment]
When you say **populate the field**, which field are you referring to?

**Comment by tomfuller2** [comment]
I think what Brent is after is that when a user adds a media file to a study record the option to select a file on the server. It wouldn't be a drop down but a browse selector from a server file, rather than a local file on the admin's computer. Sort of like JCE when you have the insert image dialogue you can choose a little icon on the upper right that allows you to insert a file from the server.

**Comment by genu** [comment]
O ok, I understand now.

**Comment by bcordis** [comment]
Yes. Just trying to get a good way to show the local server files to be added to the mediafile table.

**Comment by genu** [comment]
The **servers** workflow, I think fall into the same category as #148. Users can have a local server, remote FTP server, media from third party services (youtube, vimeo, etc), Amazon S3, etc.

Ideally this should be more modular to make it simpler for the user.

I think we should change this issue to refactor the way that servers are managed, and push it off to JBS 8.2.

The other option is to actually implement this particular feature with the current code base in the best way possible so that it can be delivered with JBS 8.1.

What do you think?

**Comment by bcordis** [comment]
Yes in a way that do as we fix up the new system.

On Wednesday, April 10, 2013, Eugen Istoc wrote:

> The _servers_ workflow, I think fall into the same category as #148https://github.com/Joomla-Bible-Study/Joomla-Bible-Study/issues/148.
> Users can have a local server, remote FTP server, media from third party
> services (youtube, vimeo, etc), Amazon S3, etc.
> 
> Ideally this should be more modular to it simpler for the user.
> 
> I think we should change this issue to refactor the way that servers are
> managed, and push it off to JBS 8.2.
> 
> The other option is to actually implement this particular feature with the
> current code base in the best way possible so that it can be delivered with
> JBS 8.1.
> 
> What do you think?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Joomla-Bible-Study/Joomla-Bible-Study/issues/71#issuecomment-16216123
> .

## 

Thanks,
Brent Cordis
2406 Hermitage Park Dr.
Hermitage, TN 37076
615.429.6426

**Comment by bcordis** [comment]
In Joomla 4, we are now moving to the Media Manager to handle this. We will be implementing this in the new version.
